ansible/roles/kubernetes/tasks/secrets.yml

```diff
@@ -80,16 +80,39 @@
     - restart daemons
 
 - include: gen_tokens.yml
-  when: inventory_hostname == groups['masters'][0]
 
-- name: Copy tokens to other masters
-  synchronize:
-    src: /etc/kubernetes/tokens/
-    dest: /etc/kubernetes/tokens
+- name: Slurp known_tokens.csv from first master
+  slurp:
+    src: "{{ kube_token_dir }}/known_tokens.csv"
+  register: known_tokens
   delegate_to: "{{ groups['masters'][0] }}"
-  when: inventory_hostname in groups['masters'] and inventory_hostname != groups['masters'][0] 
+  run_once: true
+  when: inventory_hostname in groups['masters'] and inventory_hostname == groups['masters'][0]
```

Comment:
You have use `delegate_to` already.
And i guess file `known_tokens.csv` in other masters didn't get the right content.